packages/steps/src/BuildStep.ts

```diff
@@ -308,17 +308,14 @@ export class BuildStep extends BuildStepOutputAccessor {
 
     let ifCondition = this.ifCondition;
 
-    if (ifCondition.startsWith('${') && ifCondition.endsWith('}')) {
+    if (ifCondition.startsWith('${{') && ifCondition.endsWith('}}')) {
```

Comment:
do we allow for `${{ sth }}` pattern in other places like inside of `run`? If we don't it's strange to support it only for `if`